blocks/HARDWARE/ROBOTICS/ARMS/MECADEMIC/MOVE_JOINT/MOVE_JOINT.py

```diff
@@ -17,11 +18,13 @@ def MOVE_JOINT(
     """
     Linearly move the robot's tool to an absolute Cartesian position.
 
+    Inputs
+    ------
+    ip_address: TextBlob
+        The IP address of the robot arm.
```

Comment:
We don't do the whole separate Inputs and Parameters sections thing anymore, you can just put it all under "Parameters"